LibraryComponents/Classes/AndesCoachmark/Interactors/AndesCoachMarkHighlightInteractor.swift

```diff
@@ -36,14 +37,22 @@ class AndesCoachMarkHighlightInteractor {
         let margin = self.margin + 4
         let rectConverted = view.convert(view.bounds, to: overlayView)
 
+        //If the highlightRect dosen't fit in bodyView, it will be cutted
+        //Horizontal excess
+        let rectConvertedCuttedX = max(rectConverted.minX, margin)
+        var rectConvertedCuttedWidth = rectConverted.width
+        if rectConverted.minX < margin { rectConvertedCuttedWidth -= (margin - rectConverted.minX) }
+        if rectConverted.maxX + margin > overlayView.bounds.width { rectConvertedCuttedWidth -= (margin + rectConverted.maxX - overlayView.bounds.width) }
+
+        //Vertical excess
         if !bodyViewBounds.contains(rectConverted.insetBy(dx: bodyViewBounds.minX, dy: -margin)) {
             if rectConverted.isAbove(of: bodyViewBounds) {
-                return CGRect(x: rectConverted.minX, y: bodyViewBounds.minY + margin, width: rectConverted.width, height: rectConverted.maxY - bodyViewBounds.minY - margin)
+                return CGRect(x: rectConvertedCuttedX, y: bodyViewBounds.minY + margin, width: rectConvertedCuttedWidth, height: rectConverted.maxY - bodyViewBounds.minY - margin)
             } else {
-                return CGRect(x: rectConverted.minX, y: rectConverted.minY, width: rectConverted.width, height: bodyViewBounds.maxY - rectConverted.minY - margin)
+                return CGRect(x: rectConvertedCuttedX, y: rectConverted.minY, width: rectConvertedCuttedWidth, height: bodyViewBounds.maxY - rectConverted.minY - margin)
             }
         } else {
-            return rectConverted
+            return CGRect(x: rectConvertedCuttedX, y: rectConverted.origin.y, width: rectConvertedCuttedWidth, height: rectConverted.height)
         }
     }
 
```

Comment:
```suggestion

    private func isContainedVertically(_ rect: CGRect, in container: CGRect) -> Bool {
        if rect.minY < container.minY || rect.maxY > container.maxY {
            return false
        }
        
        return true
    }
    
```

Agregamos esta función para chequear si un `CGRect` esta dentro de otro pero solamente verificando en la dirección vertical

Reply:
aca estoy agregando la funcion abajo del codigo que aparece comentado